backend/core/src/listings/listings.controller.ts

```diff
@@ -36,17 +37,19 @@ import { ValidationsGroupsEnum } from "../shared/types/validations-groups-enum"
 import { PaginationQueryParams } from "../shared/dto/pagination.dto"
 import { clearCacheKeys } from "../libs/cacheLib"
 
-export class ListingsListQueryParams extends PaginationQueryParams {
+export class ListingsQueryParams extends PaginationQueryParams {
   @Expose()
   @ApiProperty({
-    type: String,
-    example: "Fox Creek",
+    name: "filter",
     required: false,
-    description: "The neighborhood to filter by",
+    type: [String],
+    items: {
+      $ref: getSchemaPath(ListingFilterParams),
+    },
+    example: { $comparison: ["=", "<>"], status: "active", name: "Coliseum" },
```

Comment:
I take it at least for now we're going to continue using the `filter[$comparison]=<>&filter[name]=Coliseum` style? Can we start a conversation with Exygy about this? It doesn't feel very user-friendly/intuitive.

Reply:
Once I realized that it's just object syntax it made sense, but I agree that it's confusing (and that URLs are a part of UX). Fwiw, it's a standard serialization format supported by Open API called Deep Object: https://swagger.io/docs/specification/serialization/

I asked [here](https://github.com/bloom-housing/bloom/discussions/1400#discussioncomment-1002553) about their future plans.

Reply:
But for now, yes, I figured we'd leave it this way.